subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/tasks/ReleaseNeeded.java

```diff
@@ -28,11 +33,24 @@ boolean releaseNeeded(ReleaseNeededTask task, EnvVariables envVariables) {
         boolean releaseNeeded = releaseNeed.needed;
         String message = releaseNeed.explanation;
 
-        if (!releaseNeeded && task.isExplosive()) {
-            throw new GradleException(message);
-        } else {
-            LOG.lifecycle(message);
+        File releaseNeededFile = new File(task.getProject().getBuildDir(), RELEASE_NEEDED_FILENAME);
+
+        if (releaseNeededFile.exists()) {
+            releaseNeededFile.delete();
+        }
+
+        if (releaseNeeded) {
+            try {
```

Comment:
Use IOUtil.writeFile() and save 8 lines of code :)

Reply:
:+1: , thx for this hint!